app/src/test/java/org/example/app/PostgresTestContainer.java

```diff
@@ -0,0 +1,37 @@
+package org.example.app;
+
+import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.testcontainers.containers.PostgreSQLContainer;
+
+public class PostgresTestContainer implements BeforeAllCallback, AfterAllCallback {
+    private PostgreSQLContainer<?> postgres;
+
+    @Override
+    public void beforeAll(ExtensionContext context) {
+        postgres = new PostgreSQLContainer<>("postgres:14.11")

```

Comment:
Please, use the latest version `16.2-alpine3.19` https://hub.docker.com/_/postgres